eventio/base.py

```diff
@@ -88,33 +96,31 @@ def check_size_or_raise(data, expected_length, zero_ok=True):
         raise EOFError('File seems to be truncated')
 
 
-def read_next_header_toplevel(byte_stream):
-    '''Read the next toplevel header object from the file
-    Assumes position of `byte_stream` is at the beginning of a new header.
+def read_sync_marker(byte_stream):
```

Comment:
When looking at the call to this function in line 56, I first did not really get why we are calling the function but ignoring the return value. I think, this call is more like a "check", right? The function checks if we are compatible to the file and raises if not. It does so by simply reading the sync_marker, hence the name `read_sync_marker`. But still could we make the authors intention a bit more obvious somehow? Maybe something like: `raise_if_sync_marker_not_okay`? ... dunno